src/pandablocks_ioc/_hdf_ioc.py

```diff
@@ -313,13 +313,18 @@ def handle_data(self, data: HDFReceived):
 
 @dataclass
 class Dataset:
+    """A dataset name and capture mode"""
```

Comment:
What is a "Dataset" in this context? I don't know enough about this area to know if this is a commonly known topic, or a concept created here for our code.

Reply:
It's more of an abstraction - we have the `BLOCK_NAME:DATASET` record for using an alternative column label in the hdf5 file and the `BLOCK_NAME:CAPTURE` record for specifying what is captured. `class Dataset` functions as a more readable way to keep these together in the dict given from `ioc.py`.